src/docfx/lib/log/Error.cs

```diff
@@ -24,14 +24,17 @@ internal class Error
 
         public bool PullRequestOnly { get; }
 
+        public object[] OverrideParameters { get; }
+
         public Error(
             ErrorLevel level,
             string code,
             string message,
```

Comment:
Change the message type to [`FormattableString`](https://docs.microsoft.com/en-us/dotnet/api/system.formattablestring?view=netcore-3.1) and you get access to the format string and parameters.
```suggestion
            FormattableString message,
```

Reply:
OK.